resources/units/ground_units/CV9040.yaml

```diff
@@ -8,5 +8,4 @@ origin: Sweden
 price: 10
 role: Infantry Fighting Vehicle
 variants:
-  Strf 9040 IFV: {}
-  CV9040 IFV: {}
```

Comment:
Recommend reverting all changes to existing yaml files. The only thing that may need to be adjusted is the filename itself, i.e. to match the pydcs-id of the unit, but the variants should remain unchanged 😬